src/Microsoft.AspNet.Mvc.TagHelpers/LabelTagHelper.cs

```diff
@@ -32,34 +33,31 @@ public class LabelTagHelper : TagHelper
         /// <remarks>Does nothing if <see cref="For"/> is <c>null</c>.</remarks>
         public override async Task ProcessAsync(TagHelperContext context, TagHelperOutput output)
         {
-            if (For != null)
+            var tagBuilder = Generator.GenerateLabel(ViewContext,
+                                                        For.ModelExplorer,
```

Comment:
spacing, just put all parameters at 1 indent (hit enter before ViewContext and wherever it lands is good). This way it's easier to maintain.